.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Release
+on:
+  push:
+    branches:
+      - 'release/*'
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  release:
+    if: startsWith(github.ref, 'refs/tags/')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '10.x'
+          registry-url: 'https://registry.npmjs.org'
+      - name: Build artifacts
+        run: |
+          yarn install
+          yarn tsc
```

Comment:
📣 Looks like `rebuild` is never called neither is `gen`, aren't they needed? Especially that you removed the `dist` folder from the repo (from what you said in the PR description) then I am guessing you need to be sure you do them.

Reply:
From the `package.json`

```json
"rebuild": "rimraf dist/ && tsc"
```

This means that `yarn tsc` actually generates the artifacts. We also don't need to remove the folder since it won't exist in the repository anymore. Given that `yarn tsc` is sufficient.

Reply:
oh my god I just understood what the hell `rimraf` is. I thought it was some magic tool that generated files!

Reply:
Haha, yep I was also so confused in the beginning.